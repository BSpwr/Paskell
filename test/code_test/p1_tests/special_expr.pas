program SpecialExprTest;
var
    val: integer = 1337;

begin

    writeln(sqrt(val));
    writeln(sin(val));
    writeln(cos(val));
    writeln(ln(val));
    writeln(exp(val));

end.