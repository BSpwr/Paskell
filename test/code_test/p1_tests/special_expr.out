36.565010597564445
-0.9683343651587963
0.2496568790410366
7.198183577101943
Infinity
